source/_integrations/ring.markdown

```diff
@@ -49,7 +51,9 @@ This integration does NOT allow for live viewing of your Ring camera within Home
 
 ## Binary sensor
 
-Once you have enabled the [Ring integration](/integrations/ring), you can start using a binary sensor. Currently, it supports doorbell, external chimes and stickup cameras.
+The binary sensor switches between off and on when motion, doorbell rings, and intercom unlock events occur.
+
+The binary sensor is being replaced with the event entity and you should migrate any automations to the event entity by release 2025.3.0.
```

Comment:
We should add a followup PR to add a repair issue to notify them about this and tell them which automations/scripts are using it.

example to find them https://github.com/home-assistant/core/blob/20600123f8b2f6fabe4d89ae866bc467dc24c04d/homeassistant/components/unifiprotect/migrate.py#L40

Reply:
Looks like its already implemented in the other PR https://github.com/home-assistant/core/blob/dev/homeassistant/components/ring/entity.py#L41